src/utils.coffee

```diff
@@ -14,16 +14,16 @@ exports._buildOptions = _buildOptions = (config) ->
   if poolSize?
     console.warn('''
       The `poolSize` config param is deprecated.
-      Use `options: { server: { poolSize: poolSize} }` instead.
+      Use `options: { poolSize: poolSize }` instead.
```

Comment:
so options.poolSize is back to being the standard?

Reply:
> so options.poolSize is back to being the standard?

It's now been in three (!!) places.

1. poolSize was a mongodb-migrations option originally
2. then he recommended people put it in options.server.poolSize
3. then mongodb deprecated that and moved it to options.poolSize

We don't use it anywhere!